ingestion/tests/unit/topology/database/test_glue.py

```diff
@@ -0,0 +1,381 @@
+    ),
+)
+
+MOCK_DATABASE_SCHEMA = DatabaseSchema(
+    id="2aaa012e-099a-11ed-861d-0242ac120056",
+    name="default",
+    fullyQualifiedName="glue_source.118146679784.default",
+    displayName="default",
+    description="",
+    database=EntityReference(
+        id="2aaa012e-099a-11ed-861d-0242ac120002",
+        type="database",
+    ),
+    service=EntityReference(
+        id="2aaa012e-099a-11ed-861d-0242ac120002",
+        type="database",
+    ),
+)
+
+MOCK_DATABASE_PAGINATOR = {
+    "DatabaseList": [
+        {
+            "Name": "default",
+            "CreateTime": datetime.datetime(2022, 4, 13, 14, 9, 17),
+            "CreateTableDefaultPermissions": [
+                {
+                    "Principal": {
+                        "DataLakePrincipalIdentifier": "IAM_ALLOWED_PRINCIPALS"
+                    },
+                    "Permissions": ["ALL"],
+                }
+            ],
+            "CatalogId": "118146679784",
+        },
+        {
+            "Name": "mydatabase",
+            "CreateTime": datetime.datetime(2022, 4, 13, 14, 9, 17),
+            "CreateTableDefaultPermissions": [
+                {
+                    "Principal": {
+                        "DataLakePrincipalIdentifier": "IAM_ALLOWED_PRINCIPALS"
+                    },
+                    "Permissions": ["ALL"],
+                }
+            ],
+            "CatalogId": "118146679784",
+        },
+        {
+            "Name": "testdatalake_db",
+            "LocationUri": "s3://awsdatalake-testing/data",
+            "CreateTime": datetime.datetime(2022, 6, 6, 17, 19, 55),
+            "CreateTableDefaultPermissions": [
+                {
+                    "Principal": {
+                        "DataLakePrincipalIdentifier": "IAM_ALLOWED_PRINCIPALS"
+                    },
+                    "Permissions": ["ALL"],
+                }
+            ],
+            "CatalogId": "118146679784",
+        },
+        {
+            "Name": "zipcode-db",
+            "LocationUri": "s3://datalake-openmetadata-ohio/zipcode",
+            "CreateTime": datetime.datetime(2022, 4, 5, 12, 17, 53),
+            "CreateTableDefaultPermissions": [
+                {
+                    "Principal": {
+                        "DataLakePrincipalIdentifier": "IAM_ALLOWED_PRINCIPALS"
+                    },
+                    "Permissions": ["ALL"],
+                }
+            ],
+            "CatalogId": "118146679784",
+        },
+    ],
+    "ResponseMetadata": {
+        "RequestId": "21a5c41c-da65-4531-83cb-e6d3425c4ec7",
+        "HTTPStatusCode": 200,
+        "HTTPHeaders": {
+            "date": "Tue, 26 Jul 2022 16:28:52 GMT",
+            "content-type": "application/x-amz-json-1.1",
+            "content-length": "1062",
+            "connection": "keep-alive",
+            "x-amzn-requestid": "21a5c41c-da65-4531-83cb-e6d3425c4ec7",
+        },
+        "RetryAttempts": 0,
+    },
+}
+
+MOCK_TABLE_PAGINATOR = {
```

Comment:
can we move this data in a json file? 🤔

Reply:
yes

Reply:
@ulixius9 I have a `datetime` object in my dictionary. Is it possible to convert that into json?

<img width="560" alt="image" src="https://user-images.githubusercontent.com/28966081/181465596-e13147a1-7e8c-4984-83d8-e51842864b1d.png">